docs/using-scylla/cassandra-compatibility.rst

```diff
@@ -2,51 +2,63 @@
+* |v| - Available in ScyllaDB and compatible with Apache Cassandra.
+* |x| - Not available in ScyllaDB.
+* **NC** - Available in ScyllaDB, but not compatible with Apache Cassandra.
 
 Interfaces
 ----------
 
-==============================  =====================================================  =============
-Apache Cassandra                Version                                                Comments
-==============================  =====================================================  =============
-CQL                             CQL spec |cql-version|, Native protocol v4 (C* 3.11)   More below
-------------------------------  -----------------------------------------------------  -------------
-Thrift                          Compatible with Cassandra 2.1
-------------------------------  -----------------------------------------------------  -------------
-SSTable format (all versions)   3.11(mc / md ), 2.2(la), 2.1.8 (ka)                    md format is supported in Scylla 4.3, 2021.1.0 and later
-------------------------------  -----------------------------------------------------  -------------
-JMX                             3.11                                                   More Below
-------------------------------  -----------------------------------------------------  -------------
-Configuration (cassandra.yaml)	3.11
-------------------------------  -----------------------------------------------------  -------------
-Log                             NC
-------------------------------  -----------------------------------------------------  -------------
-Gossip and internal streaming	NC
-------------------------------  -----------------------------------------------------  -------------
-SSL                             NC
-==============================  =====================================================  =============
+.. list-table::
+   :widths: 25 45 30
+   :header-rows: 1
+
+   * - Apache Cassandra Interface
+     - Version Supported by ScyllaDB
+     - Comments
+   * - CQL
+     - | Fully compatible with version 3.3.1, with additional features from later CQL versions (for example, :ref:`Duration type <durations>`).
+       | Fully compatible with protocol v4, with additional features from v5.
+     - More below
+   * - Thrift 
+     - Compatible with Cassandra 2.1
+     - 
+   * - SSTable format (all versions)
+     - 3.11(mc / md ), 2.2(la), 2.1.8 (ka)
+     - ``md`` format is supported in ScyllaDB Open Source 4.3 and later, and ScyllaDB Enterprise 2021.1.0 and later
+   * - JMX   
+     - 3.11
+     - More below
+   * - Configuration (cassandra.yaml)
+     - 3.11
+     - 
+   * - Log
+     - NC
+     - 
+   * - Gossip and internal streaming
+     - NC
+     - 
+   * - SSL
+     - NC
+     - 
+
 
 ..  _3.3.1: https://github.com/apache/cassandra/blob/cassandra-2.2/doc/cql3/CQL.textile#changes
 
 Supported Tools
 ---------------
-(based on Apache Cassandra 3.11 release)
```

Comment:
Why remove this comment?
We did base the tools we package (nodetool, etc.) on some Cassandra's version, and it means we may not have features from newer Cassandra versions.

Reply:
I removed it because:
- The information that ScyllaDB is compatible with Cassandra 3.11 (+extra stuff)  is already mentioned in the introduction, and I'd like to avoid duplication. 
- Every time we add a new tool, it is added to the list of tools on that page (technically, it's a different page included on the compatibility page), but I'm not sure they are all based on 3.11. If they aren't, such a disclaimer may be misleading.
- Even if it is not misleading, I don't see any value in adding the information here.

If you still think the information here may benefit the user, please let me know, and I'll restore it.

Reply:
@nyh What do you think?

Reply:
The thing is, the "tools" (nodetool, cqlsh, etc.) are separate programs. It's a separate question of what Cassandra version Scylla itself is compatible with - and what Cassandra version its separate tools are based on.

I think - but again I'm not the expert here - I think @avelanarius would know better - that we really did take a specific version of these tools and made only minor improvements to them - unlike Scylla itself which is completely rewritten code maintaining compatibility.

@avelanarius 's commit 801f7848437a926169f263987970c94b5eeb87cf titled "Sync with Cassandra 3.11.13" suggests that, yes, our tools are based specifically on Cassandra 3.11.13. I think it's worth mentioning this somewhere in the documentation.

Reply:
@nyh I see, thanks for the explanation. I've restored the Cassandra version in that section.